### DISCUSSION TITLE
Extension Installation

### DISCUSSION THREAD

**Discussion by udarakasun** [asg017/sqlite-vss#57] [General]
Can anyone tell me how to add this extension to SQLite?

**Comment by asg017** [comment]
@udarakasun The [Installing section in the README](https://github.com/asg017/sqlite-vss#installing) lists a few different ways to install. What have you tried?

**Reply by nthverse** [reply]
pip doesn't seem to find package.  ERROR: Could not find a version that satisfies the requirement sqlite-vss (from versions: none)
ERROR: No matching distribution found for sqlite-vss

**Reply by nthverse** [reply]
Oh I see there is no windows pip yet. my bad.
